tests/std/tests/P0024R2_parallel_algorithms_adjacent_find/test.cpp

```diff
@@ -33,8 +33,13 @@ void test_case_adjacent_find_parallel(const size_t testSize) {
     assert(less_result == tmp.begin());
     assert(ne_result == tmp.begin());
 
-    auto target = tmp.begin();
+    auto remaining_attempts = quadradic_complexity_case_limit;
+    auto target             = tmp.begin();
     for (auto next = target; ++next != tmp.end(); target = next) {
+        if (--remaining_attempts == 0) {
+            return;
+        }

```

Comment:
I observe that there's a counting issue here - for a case limit of L, only L - 1 attempts will be made. The exact number of attempts isn't especially important (and the limit is well above 1), but the variable name doesn't match the control flow. Occurs in most (but not all) files.

Doing something about this would involve a fair number of changes to control flow, so I'm OK with proceeding as-is.

Reply:
I think switching to postdecrement instead or moving this if to the end of the loop body would fix it?

Reply:
OK I believe I addressed this

Reply:
Thanks for going above and beyond! :heart_eyes_cat: